### DISCUSSION TITLE
Modifying init_vars file to run on Cedar (compute canada)

### DISCUSSION THREAD

**Discussion by dsodums** [McIntosh-Lab/tvb-ukbb#148] [Help] [answered]
I'm trying to set up the pipeline to work with the Rockland data set and have a few questions about modifying init_vars.
 
The ones below in bold are software, if I'm using singularity container on cedar and compute canada, can you share what others have had to change for software?  It looks like currently they are set from when it was running on gateway so hoping I can avoid re-inventing the wheel if others can share what they've had to change.  My guess is the memory ones are unnecessary but wanted to confirm.
 
#SETUP FSL
**export FSLDIR=/opt/fsl**                                       #TO BE MODIFIED BY USER
. $FSLDIR/etc/fslconf/fsl.sh
export FSLCONFDIR="${FSLDIR}/config"
export FSLOUTPUTTYPE="NIFTI_GZ"
**export FSLGECUDAQ=cuda.q**                                                        #TO BE MODIFIED BY USER
 
#TODO implement this in the files
**export QUEUE_STANDARD="all.q"**                                                   #TO BE MODIFIED BY USER
**export QUEUE_MORE_MEM="bigmem_16.q"**                                             #TO BE MODIFIED BY USER
**export QUEUE_MAX_MEM="bigmem_64.q"**                                              #TO BE MODIFIED BY USER
 
#SETUP AFNI
**export AFNIDIR=/opt/afni**                                                        #TO BE MODIFIED BY USER
 
#SETUP FREESURFER
**export FREESURFER_HOME=/opt/freesurfer**                                          #TO BE MODIFIED BY USER
source $FREESURFER_HOME/SetUpFreeSurfer.sh  > /dev/null
 
 
export MATLAB_COMPILER_RUNTIME="$MCROOT/$MCRV"
**# export MATLAB_COMPILER_RUNTIME="/opt/MatlabR2017b/MATLAB_Runtime/v93"**         #TO BE MODIFIED BY USER

**Comment by noahfl** [comment, marked as answer]
Yeah a lot of the messaging here needs to be updated since container users don't need to modify nearly as much. For container users, the only changes you need to make are to `AFNIDIR` and `ANTSPATH`.

Note: It seems like you're using a slightly outdated `init_vars` file. Updating your repo should give you a more current version where the FreeSurfer and MATLAB export definitions are much clearer.

**Reply by dsodums** [reply]
Oh OK good to know, that would certainly explain things! Thanks for that.
